mtz-wizard.html

```diff
@@ -36,42 +36,42 @@
         /* Determines if this wizard is active or not, used to attach/detach the steps from the DOM for data reset */
         active: {
           type: Boolean,
-          value: false
+          value: false,
         },
         /* The attribute to use to match the selection key against */
         attrForSelected: {
           type: String,
-          value: 'name'
+          value: 'name',
         },
         /* The fallback selection when nothing is selected */
         fallbackSelection: {
           type: String,
-          computed: '__computeFallbackSelection(steps, attrForSelected)'
+          computed: '__computeFallbackSelection(steps, attrForSelected)',
         },
         /* Set to true when on the first step in the wizard */
         onFirstStep: {
           type: Boolean,
           value: false,
           readOnly: true,
-          notify: true
+          notify: true,
         },
         /* Set to true when on the last step in the wizard */
         onLastStep: {
           type: Boolean,
           value: false,
           readOnly: true,
-          notify: true
+          notify: true,
         },
         /* This is a CSS selector string. If this is set, only items that match the CSS selector are selectable. */
         selectable: {
           type: String,
-          value: '[wizard-step],.wizard-step'
+          value: '[wizard-step],.wizard-step',
         },
         /* The currently selected step, null if no step is selected */
         selected: {
           type: String,
-          value: null,
-          notify: true
+          value: '0',
```

Comment:
This is a breaking change, also requires an update to the associated docs since you can never have an unselected step.

Reply:
That makes sense. The reason I put it in place is because the 2.0 and 1.x versions have different defaults. So, any ideas of which to support by default? I kind of like supporting nothing being selected. That's a good point about defaulting to having an unselected step.